.github/changelog.js

```diff
@@ -0,0 +1,116 @@
+/* eslint no-var: 0, object-shorthand: 0, prefer-template: 0 */
```

Comment:
Yeah, you're right. Only reason was because I copied it from the rocket.chat repo without changing it.
I will change it now.